src/Microsoft.Extensions.Configuration.AzureAppConfiguration/FeatureManagement/FeatureManagementKeyValueAdapter.cs

```diff
@@ -93,9 +93,17 @@ public FeatureManagementKeyValueAdapter(FeatureFilterTracing featureFilterTracin
 
                     keyValues.Add(new KeyValuePair<string, string>($"{variantsPath}:{FeatureManagementConstants.Name}", featureVariant.Name));
 
-                    if (featureVariant.ConfigurationValue != null)
+                    if (featureVariant.ConfigurationValue.ValueKind == JsonValueKind.Null)
```

Comment:
Is this special case needed? We don't do it for filter parameters.

Reply:
Related to what I mentioned above, but I think this is what makes more sense for this situation. Right now if `configuration_value` is `null` then it just sets the key to empty string, and feature management sets the value of `Variant.Configuration` to an empty string. I think it makes more sense to expect `null` as the value, and I didn't want to change anything with the json flattener.

Reply:
Discussed offline. Configuration does not differentiate null and empty string.